src/components/question/Question.tsx

```diff
@@ -0,0 +1,124 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { MenuItem } from '@mui/material'
+import Box from '@mui/material/Box'
+import Divider from '@mui/material/Divider'
+import Typography from '@mui/material/Typography'
+import FormControlLabel from '@mui/material/FormControlLabel'
+import Checkbox from '@mui/material/Checkbox'
+import CheckIcon from '@mui/icons-material/Check'
+import appPallete from '~/styles/app-theme/app.pallete'
+import DragIndicatorIcon from '@mui/icons-material/DragIndicator'
+import IconButton from '@mui/material/IconButton'
+import MoreVertIcon from '@mui/icons-material/MoreVert'
+import EditIcon from '@mui/icons-material/Edit'
+import DeleteOutlineIcon from '@mui/icons-material/DeleteOutline'
+import LibraryAddCheckOutlinedIcon from '@mui/icons-material/LibraryAddCheckOutlined'
+
+import useMenu from '~/hooks/use-menu'
+import IconTitleDescription from '~/components/icon-title-description/IconTitleDescription'
+import AppChip from '~/components/app-chip/AppChip'
+
+import { ColorEnum, Answer, TableActionFunc, QuestionCategory } from '~/types'
+import { styles } from '~/components/question/Question.styles'
+
+interface QuestionProps {
+  title: string
+  answers: Answer[]
+  text: string
+  category: QuestionCategory
+  id: string
+}
+
+const Question: FC<QuestionProps> = ({
+  title,
+  answers,
+  text,
+  category,
+  id
+}) => {
+  const { t } = useTranslation()
+  const { openMenu, renderMenu, closeMenu } = useMenu()
+
+  const onAction = async (actionFunc: TableActionFunc) => {
+    closeMenu()
+    await actionFunc(id)
+  }
+  const rowActions = [
+    {
+      label: (
+        <Box sx={styles.editIconWrapper}>
+          <EditIcon sx={styles.editIcon} />
+          {` ${t('common.edit')}`}
+        </Box>
+      ),
+      func: (id: string) => {
+        console.log('Edit', id)
+      }
+    },
+    {
+      label: (
+        <Box sx={styles.deleteIconWrapper}>
+          <DeleteOutlineIcon color='primary' sx={styles.deleteIcon} />
+          {` ${t('common.delete')}`}
+        </Box>
+      ),
+      func: (id: string) => {
+        console.log('Delete', id)
+      }
+    }
+  ]
+  const menuItems = rowActions.map(({ label, func }) => (
+    <MenuItem key={label} onClick={() => void onAction(func)}>
+      {label}
+    </MenuItem>
+  ))
+
+  const answersList = answers.map((answer, i) => (
+    <Box key={answer.text} sx={styles.answer}>
+      <FormControlLabel
+        checked={i == 1}
+        control={<Checkbox />}
+        label={answer.text}
+      />
+
+      {answer.isCorrect ? (
+        <CheckIcon sx={{ color: appPallete.basic.orientalHerbs }} />
```

Comment:
answer.isCorrect && 
move sx to style file

Reply:
fixed